index.html

```diff
@@ -8,11 +8,22 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" rel="stylesheet"/>
 </head>
 <body>
     <header>
-        <h1 id="header">Каталог ИИ-сервисов от AIA Podcast</h1>
-        <button id="lang-switch">RU/EN</button>
+        <div class="header-container">
+            <h1 id="header">Каталог ИИ-сервисов от AIA Podcast</h1>
+            <div class="settings-menu">
+                <button id="settings-button">
+                    <span class="material-symbols-outlined">menu</span>
+                </button>
+                <div class="settings-content">
+                    <button id="lang-switch">RU/EN</button>
+                    <button id="theme-toggle">Dark Mode</button>
+                </div>
```

Comment:
Instead of menu I suggest you better add single button with the theme switch icon. Like iOS switch or something